var/spack/repos/builtin/packages/wgrib2/package.py

```diff
@@ -177,6 +177,7 @@ def url_for_version(self, version):
     depends_on("ip@5.1:", when="@3.5: +ipolates")
     depends_on("lapack", when="@3.5: +ipolates")
     depends_on("libaec@1.0.6:", when="@3.2: +aec")
+    depends_on("netcdf-c", when="@3.6: +netcdf")
```

Comment:
```suggestion
    depends_on("netcdf-c", when="@3.4: +netcdf")
```
@AlexanderRichert-NOAA @AlysonStahl-NOAA I think we can simplify this, because the variant `netcdf` is only valid for version 3.4+, see lines 108-113. Or we make it explicit (and redundant) and say `@3.4: +netcdf`.

Reply:
> @AlexanderRichert-NOAA @AlysonStahl-NOAA I think we can simplify this, because the variant `netcdf` is only valid for version 3.4+, see lines 108-113. Or we make it explicit (and redundant) and say `@3.4: +netcdf`.

Unsolicited opinion: given [Alyson's comment](https://github.com/JCSDA/spack/pull/568#issuecomment-3378097683), `@3.4: +netcdf` may be redundant, but it also provides a bit of context with respect to the `wgrib2` code history itself, particularly if a comment like 

`# options to use netcdf3 or netcdf4 merged into a single option with v3.4.0
`

accompanies the change.

Reply:
I am on board with your suggestion!